tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -415,6 +471,17 @@ def get_all_test_cases():
             print('\n'.join(get_all_test_cases()))
         return 0
 
+    if args.generate_hrr_tls13_compat_tests:
+        if args.output:
+            with open(args.output, 'w', encoding="utf-8") as f:
+                f.write(SSL_OUTPUT_HEADER.format(
+                    filename=os.path.basename(args.output)))
```

Comment:
```suggestion
                f.write(SSL_OUTPUT_HEADER.format(
                    filename=os.path.basename(args.output),
                    param='-r'))
```
And change line 387 to
```
# `generate_tls13_compat_tests.py {param}`.
```
Please also change line 467
Then regenerate `tls13-compat-hrr.sh` with ``generate_tls13_compat_tests.py -r -o tests/opt-testcases/tls13-compat-hrr.sh`.